orc8r/gateway/python/scripts/register.py

```diff
@@ -107,22 +135,34 @@ def main():
             Challenge Key
             -----------
             {}
-            Control Proxy
-            -----------
-            {}
-            """,
+            """
         )
-        print(msg.format(req.hwid, req.challenge_key, res.control_proxy))
+        print(msg.format(req.hwid, req.challenge_key))
+
+        if not args.no_control_proxy:
+            save_control_proxy(res.control_proxy)
+            msg = textwrap.dedent(
+                """
+                Control Proxy
+                -----------
+                {}
+                """
+            )
+            print(msg.format(res.control_proxy))
     except Exception as e:
         msg = textwrap.dedent(" > Error: {} ")
         print(msg.format(e))
         sys.exit(1)
 
-    if not args.no_control_proxy:
-        save_override_config("control_proxy", res.control_proxy)
+    print("> Waiting {} seconds for next bootstrap".format(
+        BootstrapManager.LONG_BOOTSTRAP_RETRY_INTERVAL.total_seconds()))
+    print()
 
+    time.sleep(BootstrapManager.LONG_BOOTSTRAP_RETRY_INTERVAL.total_seconds())
```

Comment:
Idk if this is necessary, just that if we run the script without this, it'll fail. Idk if there are better alternatives though